src/targets/gpu/target.cpp

```diff
@@ -57,6 +58,8 @@ std::vector<pass> target::get_passes(migraphx::context& gctx) const
         dead_code_elimination{},
         eliminate_contiguous{},
         dead_code_elimination{},
+        adjust_allocation{},
```

Comment:
Can you remove the `adjust_allocation` pass in this PR and just have the updates for `eliminate_contiguous`?

Reply:
Wait, I guess this wont be needed if we merge PR #246.